### PR TITLE
Added command objects to wrap invocation of controllers

### PR DESCRIPTION
**Pull request by shoatman**
You've already reviewed this.  Just opening a new PR for simplicity sake post rebase on latest MSAL and common changes for webview.

### CODE REVIEW

File: msal/src/main/java/com/microsoft/identity/client/controllers/MSALInteractiveTokenCommand.java [hunk 1]
```diff
@@ -0,0 +1,49 @@
+package com.microsoft.identity.client.controllers;
```
Comment (addev-ashish) [inline msal/src/main/java/com/microsoft/identity/client/controllers/MSALInteractiveTokenCommand.java]: License is needed at the top of file, also could you please run the auto formatter on this file.

File: msal/src/main/java/com/microsoft/identity/client/controllers/MSALTokenCommand.java [hunk 2]
```diff
@@ -0,0 +1,75 @@
+package com.microsoft.identity.client.controllers;
+
+import android.content.Context;
+import android.content.Intent;
+
+import com.microsoft.identity.client.AuthenticationCallback;
+import com.microsoft.identity.client.AuthenticationResult;
+
+public class MSALTokenCommand implements MSALTokenOperation {
```
Comment (addev-ashish) [inline msal/src/main/java/com/microsoft/identity/client/controllers/MSALTokenCommand.java]: License is needed at the top of file, also could you please run the auto formatter on this file.

File: msal/src/main/java/com/microsoft/identity/client/controllers/MSALTokenOperation.java [hunk 3]
```diff
@@ -0,0 +1,10 @@
+package com.microsoft.identity.client.controllers;
```
Comment (addev-ashish) [inline msal/src/main/java/com/microsoft/identity/client/controllers/MSALTokenOperation.java]: License is needed at the top of file, also could you please run the auto formatter on this file.

File: msal/src/main/java/com/microsoft/identity/client/controllers/MSALInteractiveTokenCommand.java (hunk 1 above)
Comment (shoatman) [reply msal/src/main/java/com/microsoft/identity/client/controllers/MSALInteractiveTokenCommand.java]: Done.  Added to all files.

File: msal/src/main/java/com/microsoft/identity/client/controllers/MSALTokenCommand.java (hunk 2 above)
Comment (shoatman) [reply msal/src/main/java/com/microsoft/identity/client/controllers/MSALTokenCommand.java]: added to all files.

File: msal/src/main/java/com/microsoft/identity/client/controllers/MSALTokenOperation.java (hunk 3 above)
Comment (shoatman) [reply msal/src/main/java/com/microsoft/identity/client/controllers/MSALTokenOperation.java]: Done.

**Pull request merged**